### DISCUSSION TITLE
Add 'loading observations' message

### DISCUSSION THREAD

**Discussion by aliya-ryan** [digidem/mapeo-mobile#789] [Ideas]
Currently when you open Mapeo fresh, if you look straight away at the observations list, there is a message saying "No Observations" which is alarming if you think you have collected some data. What is really happening is that Mapeo is still loading the observations, and if you look back a little while later the observations appear as normal.

It would be better if the app could distinguish between 'no observations' and 'loading observations' and tell the user this, to reduce anxiety and confusion.

Thanks

**Comment by gmaclennan** [comment]
I started work on this in December but it needs more done. Tracking in PR #395
